src/components/include/test/policy/mock_policy_manager.h

```diff
@@ -33,24 +33,17 @@
 #ifndef SRC_COMPONENTS_INCLUDE_TEST_POLICY_MOCK_POLICY_MANAGER_H_
 #define SRC_COMPONENTS_INCLUDE_TEST_POLICY_MOCK_POLICY_MANAGER_H_
 
-#include <string>
```

Comment:
@VProdanov , i deleted it because all of this headers must be included in inherited class.